src/Common/Core/MorganStanley.ComposeUI.Common.Interfaces/SyncPriority.cs

```diff
@@ -0,0 +1,73 @@
+﻿/// ********************************************************************************************************
```

Comment:
Yes, this is a copy paste of a small chunk of avalonia code, because I did not want a non-visual project to be dependent on Avalonia visual dll - this is an important principle when working with WPF or Avalonia. So I simply copied the file. I think it should be fine. If needed I can rewrite this file line by line, changing the case to underscore. 
Avalonia comes under the most permissive MIT license, so I believe it should be ok even for the final product. 

But again - we are dealing with a project at a very early stage, let us put a mental check on it and let it be for now. If needed we'll come back and change it.

Reply:
remove the code and the interface